pkg/backend/apply.go

```diff
@@ -175,9 +181,17 @@ func confirmBeforeUpdating(kind apitype.UpdateKind, stack Stack,
 
 		choices := []string{string(yes), string(no)}
 
+		// explain is down here instead of with the other choices so we can optionally emit an emoji
+		explain := "explain " + cmdutil.EmojiOr("✨", "")
+
 		// For non-previews, we can also offer a detailed summary.
 		if !opts.SkipPreview {
 			choices = append(choices, string(details))
+
+			// If we have an explainer (pulumi-cloud) we can offer to explain the changes.
+			if explainer != nil && opts.Display.ShowCopilotExplain {
```

Comment:
This doesn't work, ho hum, opts copying stuff again..

Reply:
"Fixed" in https://github.com/pulumi/pulumi/pull/19138/commits/1f0acfacd9f4cc31a6e4527b1b74ab053ba2ebfc